dynamodb_feature_store.js

```diff
@@ -138,68 +134,60 @@ function dynamoDBFeatureStoreInternal(tableName, options) {
     });
   };
 
-  this.initializedInternal = function(cb) {
+  store.initializedInternal = function(cb) {
+    var token = initializedToken();
     dynamoDBClient.get({
       TableName: tableName,
-      Key: initializedToken,
+      Key: token,
     }, function(err, data) {
       if (err) {
         logger.error(err);
         cb(false);
         return;
       }
-      var inited = data.Item && data.Item.key === initializedToken.key;
+      var inited = data.Item && data.Item.key === token.key;
       cb(!!inited);
     });
   };
 
-  this.close = function() {
+  store.close = function() {
     // The node DynamoDB client is stateless, so close isn't a meaningful operation.
   };
 
-  this.batchWrite = function(ops) {
```

Comment:
This was being exposed as a property on `this` just so that the test code could use it. I moved it into `dynamodb_helpers` instead.